config/webhook/manifests.yaml

```diff
@@ -26,3 +26,24 @@ webhooks:
     resources:
     - edgedeployments
   sideEffects: None
+- admissionReviewVersions:
```

Comment:
can we reuse the same service for both kinds and unified it so the only changes to this file is:
1. Generalize the name of the path of the service in line 15 (e.g. s//validate-management-project-flotta-io-v1alpha1-edgedeployment//validate-management-project-flotta-io-v1alpha1)
2. Extend the rules to refer to edgedevices as well

Reply:
any particular reason for reusing it? the different section will be needed if we would want various config for each.

Reply:
no, let's keep it separated.